doc/src/index.rst

```diff
@@ -21,31 +21,42 @@ This is the central page for all of SymPy's documentation.
 :ref:`SymPy Tutorials <tutorial>`
 -------------------------------------------------------
 
-*Master SymPy concepts and features*
+Tutorials introduce Sympy features to people who are new to the project. It includes
+installation instructons, a basic introduction to Sympy operations and common gotchas.
 
 
 ..  rst-class:: column
 
 :ref:`How-to Guides <guides>`
 -------------------------------------------------------
 
-*Step-by-step intructions on how to do different key developer tasks*
+These are step-by-step intructions on how to do different key developer tasks. It details
+instructions on installing SymPy from source for development. The development setup and workflow
+is also discussed with elaborate details on debugging, building the docs, and general guidelines
+on wrting documentation and docstrings.
 
 
 ..  rst-class:: column clearfix
 
 :ref:`Explanation <explanation>`
 -------------------------------------------------------
 
-*Common pitfalls and advanced topics*
+Intermediate SymPy users and developers should reference this category of documentation
+for common pitfalls and advanced topics. The core SymPy features like variables, functions,
+symbolic expressions are detailed but it does not completely exhaust all SymPy operations.
+Other special topics, namely, **finite differential approximations to derivative** and
+**classification of SymPy objects** are also discussed.
```

Comment:
We shouldn't list the articles that are in this section here. This section will be expanded to include other stuff in the future (the very near future as this is what I will be working on for my CZI grant project).